src/index.html

```diff
@@ -8,7 +8,224 @@
+                <h6>2010 - 2013</h6>
+                <p>description_2</p>
+              </div>
+              <div>
+                <h6>Finance Director@Computers & More LTD.</h6>
+                <h6>2008 - 2012</h6>
+                <p>description_3</p>
+              </div>
+            </div>
+            <!--Slider Education-->
+            <div id="Education_Container">
+              <div id="Education_title">
+                <h2 style="text-transform:uppercase">education</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <h6>The San Francisco Business School</h6>
+                <h6>2010 - 2012</h6>
+                <p>description_1</p>
+              </div>
+              <div>
+                <h6>NYC School of Marketing and Advertising</h6>
+                <h6>2007 - 2010</h6>
+                <p>description_2</p>
+              </div>
+              <div>
+                <h6>L.A. University of Applied Sciences</h6>
+                <h6>2002 - 2006</h6>
+                <p>description_3</p>
+              </div>
+            </div>
+            <!--Slider Hobbies-->
+            <div id="Hobbies_Container">
+              <div id="Hobbies_title">
+                <h2 style="text-transform:uppercase">Hobbies & intreset</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <ul>
+                  <li style="display:inline-block">
+                    <img src="cinema.jpg" alt="icon1">
+                    <p>Cinema</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Sport.jpg" alt="icon1">
+                    <p>Sport</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Theatre.jpg" alt="icon1">
+                    <p>Theatre</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="BBQ.jpg" alt="icon1">
+                    <p>BBQ</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Party.jpg" alt="icon1">
+                    <p>Party</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Cooking.jpg" alt="icon1">
+                    <p>Cooking</p>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+
+          <!-- Skill -->
+          <div id="skill">
+            <div id="skill_title">
+              <h2 style="text-transform:uppercase">skill</h2>
+            </div>
+            <div id="skill_container">
+              <div id="block1">
```

Comment:
`id` and `class` are not the part of lesson one, I am glad to know that you've got some knowledge about them. But it seems that you tend to overuse the `id` property. We will discuss the occasions when we should choose `id and when class in the lesson 2. Hopefully, you will get a better understanding then.